src/main/java/com/google/devtools/build/lib/skyframe/serialization/autocodec/BUILD

```diff
@@ -11,6 +11,10 @@ filegroup(
 java_library(
     name = "autocodec",
     exported_plugins = [":autocodec-plugin"],
+    javacopts = select({
```

Comment:
@janakdr What's your thought on building autocodec with `--add-modules=jdk.unsupported` for Java 9? Do you think there's a better way?

Reply:
I think the Java 9 changes should be made separately from the "bump checkerframework to 2.2.2 and stop building from source" change. The latter is big enough for one commit, and the details former may need more discussion.

Reply:
Done.